packages/filebrowser/src/model.ts

```diff
@@ -586,7 +586,7 @@ export class FileBrowserModel implements IDisposable {
 
     // If either the old value or the new value is in the current path, update.
     if (value) {
-      this._scheduleUpdate();
+      this._poll.refresh();
```

Comment:
Should this be `await this._poll.refresh();` or `void this._poll.refresh();`?

Reply:
I'm assuming it should be `void`ed.

Reply:
Since it isn't an async function, I think `void` is what is expected - fire and forget the update.

Reply:
Yep, good catch.